packages/itwin/grouping-mapping-widget/src/widget/components/Grouping.tsx

```diff
@@ -75,7 +75,7 @@ const fetchGroups = async (
       iModelId,
       mappingId,
     );
-    setGroups(groups);
+    setGroups(groups.sort((a, b) => a.groupName?.localeCompare(b.groupName ?? "") ?? 1));
```

Comment:
`Group.groupName` from `@itwin/insights-client` should never be null or undefined.